tests/integration/components/public/event-map-test.js

```diff
@@ -0,0 +1,26 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('public/event-map', 'Integration | Component | public/event map', {
+  integration: true
+});
+
+test('it renders', function(assert) {
+  assert.expect(1);
+
+  this.render(hbs`{{public/event-map event=model.event}}`);
+  assert.ok(this.$());
+
+});
+
+test('it includes the marker', function(assert) {
```

Comment:
This is not needed since this is an external component. This would be rendered when the above renders.